wca-regulations.md

```diff
@@ -324,6 +324,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 12c1) Moves are applied with one of the two smallest surfaces of the equatorial slice on the left side of the front face.
     - 12c2) (x, y) means: turn upper layer x times 30 degrees clockwise, turn bottom layer y times 30 degrees clockwise.
     - 12c3) "/" means: turn the right half of the puzzle 180 degrees.
+    - 12c4) Metric for Square-1: (x, y) counts as one move, "/" counts as one move.
```

Comment:
Uber nit: Do we care about `(0, 0)`?

Reply:
We can actually make 12c2 more precise, by specifying that x and y must be in [-5; 6], and that x and y cannot be both equal to 0.

Reply:
Yes, clarifying this in 12c2 sounds good!